apps/ewallet_db/lib/ewallet_db/account.ex

```diff
@@ -339,6 +345,124 @@ defmodule EWalletDB.Account do
     )
   end
 
+  def descendant?(ancestor, descendant_id) do
+    ancestor
+    |> get_all_descendants()
+    |> Enum.map(fn descendant -> descendant.id end)
+    |> Enum.member?(descendant_id)
+  end
+
+  @spec get_all_descendants_uuids(%Account{}) :: List.t()
+  def get_all_descendants_uuids(account) do
+    query_family_uuids(account, "
+      WITH RECURSIVE accounts_cte(uuid, parent_uuid, depth) AS (
+        SELECT ta.uuid, ta.parent_uuid, $1::INT AS depth
+        FROM account AS ta WHERE ta.uuid = $2
```

Comment:
Would be nice if we use full names instead of `ta`, `c`, etc. for these cases to help with readability